lib/index.js

```diff
@@ -139,6 +139,7 @@ const defaults = {
     ignoreConflicts: false,
     domainPrefix: false,
     triggerTraceEnabled: true,
+    ec2MetadataTimeout: undefined,      // ms to wait for ec2 metadata reply
```

Comment:
where are you setting the default? 1000ms

Reply:
i don't set the default in this case. i leave it to oboe which (i thought) had the default built-in. that way i'm not duplicating oboe's default nor do i have to stay in sync with it.

the way the code works (you'd have to look into the bindings to see it) is that it will leave it the default value if a value is undefined.

Reply:
good - thanks for the explanation.